### PR TITLE
Require braces to be present for environment variable expansion.

### PR DESCRIPTION
**Pull request by andrewkroh**

### CODE REVIEW

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -13,6 +13,7 @@ https://github.com/elastic/beats/compare/v1.2.0...master[Check the HEAD diff]
 ==== Breaking changes
 
 *Affecting all Beats*
+- Require braces for environment variable expansion in config files {pull}1304[1304]
```
Comment (andrewkroh) [inline CHANGELOG.asciidoc]: Do we want these backports to point to the original PR or the backport PR?

**Comment by tsg** [comment]
LGTM.

File: CHANGELOG.asciidoc (hunk 1 above)
Comment (tsg) [inline CHANGELOG.asciidoc]: I'd say any is fine, whatever brings more context to someone reading the changelog.

**Comment by andrewkroh** [comment]
jenkins, test it

**Pull request merged**